### DISCUSSION TITLE
decrease threashold to trigger on_click events

### DISCUSSION THREAD

**Discussion by ghost** [flet-dev/flet#987] [Ideas]
deleted because i noticed that only some on_clicks act strange. investigating

**Comment by FeodorFitsner** [comment]
What Flet control are you referring to? Could you please provide a code example?

**Reply by ghost** [reply]
i just noticed that if you put an IconButton inside of a container and add the on_click to the container instead of the IconButton, fast clicks with below 100ms left mouse button downtime wont be registered/wont trigger the on_click. 
Fast Tapevents on the other hand will still trigger it.

Anyways, lessons learned by my bad design.
